### ISSUE TITLE
Odd max-columns with using phpcbf

### ISSUE THREAD

**Issue by joehoyle** [humanmade/coding-standards#62]
I think I saw some mention of a columns length change somewhere, but when I run `phpcbf` it's doing odd indenting / line returns, such as:

```php
add_filter( 'determine_current_user', function ( $user ) {
	return 1;
} );
```

=>

```php
add_filter(
	'determine_current_user', function ( $user ) {
		return 1;
	}
);
```

Like, what even is that?!

**Comment by mattheu** [comment]
We just started using `phpcbf` And noticed it was messing up some functions. I ended up just selectively disabling the following sniff from PHPCBF only:

```
    <rule ref="vendor/humanmade/coding-standards">
        <!-- Disable check from phpcbf - as its fixing things that should be allowed. -->
        <exclude phpcbf-only="true" name="PEAR.Functions.FunctionCallSignature"/>
    </rule>
```

**Comment by kadamwhite** [comment]
needs refresh, may be superseded by recent coding standards changes and/or PHPCS updates.

**Comment by kadamwhite** [comment]
Note that per @mikeselander we've un-set the WPCS rule about multiple arguments on one line so this would be focused on the line lengh issue specifically
